package_io/input_parser.star

```diff
@@ -107,57 +159,15 @@ def parse_input(input_args):
 	if result["network_params"]["capella_fork_epoch"] > 0 and result["network_params"]["electra_fork_epoch"] != None:
 		fail("electra can only happen with capella genesis not bellatrix")
 
-	required_num_validtors = 2 * result["network_params"]["slots_per_epoch"]
-	actual_num_validators = len(result["participants"]) * result["network_params"]["num_validator_keys_per_node"]
-	if required_num_validtors > actual_num_validators:
-		fail("required_num_validtors - {0} is greater than actual_num_validators - {1}".format(required_num_validtors, actual_num_validators))
+	if total_participant_count < 1:
```

Comment:
When would this be true?

Reply:
The most basic setup with participants set to 0: 
```json
{
  "participants": [
  ]
}
```
I get this error message:
```
There was an error interpreting Starlark code 
Evaluation error: index 0 out of range: empty list
        at [github.com/kurtosis-tech/eth-network-package@lporoli/ticket-161/main.star:17:67]: run
```

So it seems like that you can't run it with 0 participants.

Reply:
ok, good catch @barnabasbusa I didn't analyze the logic here, I basically transferred the validation from the eth2 package because we are calling down this function from there now and I saw this different between both functions

Reply:
I'll remove it

Reply:
This was in the previous one in case someone set only one participant and set its `count` to 0, but yeah we should just fail loudly perhaps